src/components/large-countdown-clock/LargeCountdownClock.tsx

```diff
@@ -59,7 +63,9 @@ export default function LargeCountdownClock({
   const accentColor = useForegroundColor('accent');
 
   // convert clock time to seconds
-  const totalSeconds = minutes ? minutes * 60 + seconds : seconds;
+  const mins = initialMinutes || minutes;
```

Comment:
Wouldn't this always evaluate to the constant? `ENS_SECONDS_WAIT`

Reply:
yeah `totalSeconds` would resolve to `ENS_SECONDS_WAIT` but `timeRemaining` would resolve to the seconds left, so we can see the clock animation progressing in the clock

![image](https://user-images.githubusercontent.com/12115171/166577185-e187379f-7684-474f-bf01-7bc7493da861.png)